core/api/jrpc/value_converter.hpp

```diff
@@ -207,7 +207,12 @@ namespace kagome::api {
   }
 
   inline jsonrpc::Value makeValue(const primitives::Justification &val) {
-    return makeValue(val.data);
+    jArray frnk;
+    frnk.emplace_back(jArray({'F', 'R', 'N', 'K'}));
+    frnk.emplace_back(jArray(val.data.begin(), val.data.end()));
+    jArray res;
```

Comment:
why not just to return frnk here?

Reply:
I did it like in polkadot - triple-nested array